dir.props

```diff
@@ -0,0 +1,335 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
dir.props/dir.targets should not be showing up in this PR..?

Reply:
Yes, this was by mistake hence removed.